stdlib/public/Concurrency/DiscardingTaskGroup.swift

```diff
@@ -604,9 +827,10 @@ public struct ThrowingDiscardingTaskGroup<Failure: Error> {
   @available(*, unavailable, message: "Unavailable in task-to-thread concurrency model", renamed: "addTask(operation:)")
 #endif
   @_alwaysEmitIntoClient
+  @_allowFeatureSuppression(IsolatedAny)
   public mutating func addTask(
     priority: TaskPriority? = nil,
-    operation: __owned @Sendable @escaping () async throws -> Void
+    operation: __owned @Sendable @escaping @isolated(any) () async throws -> Void
```

Comment:
maybe just not implemented yet (?), would we want to remove the `@Sendable` since isolated(any) will imply Sendable?

Reply:
It's not implemented yet, but also the sanest approach for the feature-suppression mechanism is to just decline to print `@isolated(any)` and not to try to add back `@Sendable`, so we do need to keep the redundant `@Sendable` in there.

Reply:
Gotcha, makes sense!